Apps/Src/CAN_Queue.c

```diff
@@ -0,0 +1,63 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+/** CAN_Queue.c
+ * Queue that holds all CAN messages that Task_CANBusConsumer needs to send.
+ */
+
+#include "CAN_Queue.h"
+#include "os.h"
+#include "CANbus.h"
+#include "Tasks.h"
+
+// fifo
+#define FIFO_TYPE CANMSG_t
+#define FIFO_SIZE 256
+#define FIFO_NAME CAN_fifo
+#include "fifo.h"
+
+static CAN_fifo_t canFifo;
+static OS_SEM canFifoSem;
```

Comment:
Should we have these follow the same naming system as others? Name_Sem4, Name_Mutex

Reply:
Yeah, I agree it's a good idea to keep the naming consistent. These aren't new shared resources though because they are internal to this module. The module itself is a shared resource that is replacing a different shared resource (the CAN OS queue)